src/contents/a-comprehensive-guide-to-developing-with-scroll-sdk-charts.mdx

```diff
@@ -0,0 +1,350 @@
+---
+name: "A Comprehensive Guide to Developing with Scroll SDK Charts"
+index: 12
+summary: With Scroll SDK, yoou can...
+author: Ahmet Tahir Yıldız
+authorIcon: https://x.com/LandoC4lrissian/photo 
```

Comment:
The image is not visible. We need a direct link to twitter, I think replacing it to `https://pbs.twimg.com/profile_images/1721586474858983424/N50Rpjfv_200x200.jpg` should work